core/src/main/codegen/templates/Parser.jj

```diff
@@ -1698,10 +1700,38 @@ SqlNode SqlUpdate() :
     ]
     [ [ <AS> ] alias = SimpleIdentifier() ]
 <#if parser.allowUpdateFromTable>
-    (
-        <FROM> secondTable = TableRef()
-        [ [ <AS> ] secondAlias = SimpleIdentifier() ]
-    )?
+    [
+        (
+            <FROM>
+            sourceTable = TableRef() {
```

Comment:
Since this logic is duplicated in the <COMMA> case below, can we break this out into its own parser helper function?

Reply:
It looks like it is possible, though I think then the two lists will need to be initialized to be empty before hand right?